### PR TITLE
SEO-1939-Docking-manager-docs-WinForms-Money-keywords

### PR DESCRIPTION
**Pull request by Gowri-V-L**

### CODE REVIEW

File: WindowsForms/Docking-Manager/Getting-Started.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ You can create the Windows Forms application with docking manager control as fol
 
 **Creating the project**
 
-Create a new Windows Forms project in the Visual Studio to dock panels as like Visual Studio using the docking manager.
+Create a new Windows Forms project in the Visual Studio to [dock panels](https://www.syncfusion.com/winforms-ui-controls/docking-manager) as like Visual Studio using the docking manager.
```
Comment (Girija-Rajendran) [inline WindowsForms/Docking-Manager/Getting-Started.md:39]: add platforms name before the control name

Comment (Gowri-V-L) [reply WindowsForms/Docking-Manager/Getting-Started.md:39]: changed

**Comment by Gowri-V-L** [comment]
rebuild

**Comment by Gowri-V-L** [comment]
rebuild

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**